model/src/olivia.rs

```diff
@@ -104,13 +155,25 @@ impl BitMexPriceEventId {
     pub fn digits(&self) -> usize {
         self.digits
     }
+
+    pub fn index_price(&self) -> IndexPrice {
+        self.index
+    }
+
+    pub fn contract_symbol(&self) -> ContractSymbol {
```

Comment:
nit: `as_contract_symbol`?

https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv

Reply:
It's not a conversion on `self` though. Otherwise I would agree!